### DISCUSSION TITLE
hello,i don't understand .{"_placeholder":true,"num":0} what is that mean?

### DISCUSSION THREAD

**Discussion by fcy197778770** [miguelgrinberg/python-socketio#737] [Q&A]
![image](https://user-images.githubusercontent.com/68221995/118429412-3a40a780-b704-11eb-8afe-8640e4508fab.png)
i can't Received anything and  i dont khow why
THX

**Comment by miguelgrinberg** [comment]
This is an aspect of the Socket.IO protocol. If you are interested in this level of detail, consult the Socket.IO protocol documentation.

Your binary data is in the packet that follows right after the one you highlighted.
